bin/sl-runctl.js

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env node
-// Copyright IBM Corp. 2014,2015. All Rights Reserved.
+// Copyright Sam Roberts 2014,2016. All Rights Reserved.
```

Comment:
The years are correct. It is `<year-created>[,<year-last-modified>]`. The name is wrong, of course. I'm guessing you used `slt copyright` instead of `apict copyright`.